packages/contracts/script/ConfigInitializer.sol

```diff
@@ -24,7 +24,7 @@ library ConfigInitializer {
             8, // creature index
             4, // length
             8, // width
-            60, // round interval second
```

Comment:
10 seconds is not enough at all

Reply:
For a quick test

Reply:
> For a quick test

So just keep it in your local and not add it to git